libavpipe/src/avpipe_xc.c

```diff
@@ -3623,6 +3606,12 @@ avpipe_xc(
             break;
         }
 
+        if (input_packet->flags & AV_PKT_FLAG_CORRUPT) {
+            elv_warn("packet corrupt pts=%"PRId64, input_packet->pts);
+            av_packet_free(&input_packet);
+            continue;
+        }
```

Comment:
I would prefer to put this in a separate PR - or add a flag to enable/disable.

Reply:
FFmpeg does this all the time, I am not sure why you want to disable it. BTW, we can add it later